nevergrad/optimization/optimizerlib.py

```diff
@@ -1495,7 +1501,7 @@ def _internal_ask_candidate(self) -> p.Parameter:
         # We request a bit more points than what is really necessary for our dimensionality (+dimension).
         sample_size = int((self.dimension * (self.dimension - 1)) / 2 + 2 * self.dimension + 1)
         if (
-            self._num_ask % max(13, self.num_workers, self.dimension) == 0
+            self._num_ask % max(13, self.num_workers, self.dimension, int(self.modulo * sample_size)) == 0
```

Comment:
I don't get this. can you add a docstring about it? and modulo may not be the right name if it's a float :s

Reply:
modulo_ratio ? It's the ratio between the modulo (used for deciding the frequency at which we use the metamodel) and the sample-size.

Reply:
frequency_ratio?